x/marker/types/msg_test.go

```diff
@@ -245,3 +245,18 @@ func TestMsgAddFinalizeActivateMarkerRequestValidateBasic(t *testing.T) {
 		})
 	}
 }
+
+func TestMsgSupplyIncreaseProposalRequestGetSigners(t *testing.T) {
+	authority := sdk.AccAddress("input111111111111111")
+	targetAddress := sdk.AccAddress("input111111111111111")
+	amount :=
+		sdk.Coin{
+			Amount: math.NewInt(100),
+			Denom:  "",
```

Comment:
Safest to give a denom here in the off chance that not having one causes an unrelated/unexpected problem.